src/components/graph/GraphBehavior.ts

```diff
@@ -24,8 +22,5 @@ export class GraphBehavior implements IGraphBehavior {
       zoomRef.current?.scaleTo(zoomSelection, 1);
       zoomRef.current?.translateTo(zoomSelection, 0, 0, [0, 0]);
     };
-    if (disableScrollToZoom) {
```

Comment:
Update reason: This class is for lib consumers to control the graph from outside. This piece of logic is not for that responsibility.